.ci/helper_shellchecks.sh

```diff
@@ -10,8 +10,13 @@ mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash
 SHELLSCRIPT_ERROR=0
 
 for shellscript in "${shellscript_locations[@]}"; do
-    echo -e "${ANSI_GREEN}Running shellcheck on ${shellscript}"
-    shellcheck "${shellscript}" || SHELLSCRIPT_ERROR=1
+    # These two scripts are running on the device, so they should use pure sh
```

Comment:
Ah, it looks like I have misunderstood the "# shellcheck disable" statement.

Thank you for the suggestion, I am not good at bash / sh :(. I will update the code.